__tests__/app.test.js

```diff
@@ -531,3 +531,22 @@ describe('GET /api/articles (topics)', () => {
         })
     });
 })
+describe('GET /api/articles/:article_id (comment_count)', () => {
+    it('GET: 200 - Should include comment_count property with correct type in response object when given valid article_id', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => { 
+            expect(body.article).toHaveProperty('comment_count')
+            expect(typeof body.article.comment_count).toBe('number')
+        })
+    });
+    it('GET: 200 - Should return the correct number of comments from an article', () => {
```

Comment:
I think that, really, this could have been combined with the test above. If it passes the test that the comment count is 2, then it's fair to say that it does indeed have a comment count property and that it's value is a number.

Reply:
I 100% agree, just felt wrong to only push one test...I nearly typed out all the earlier tests, but thought surely not.